docs/resources/sso_settings.md

```diff
@@ -255,6 +255,7 @@ Optional:
 - `allow_sign_up` (Boolean) Whether to allow new Grafana user creation through SAML login. If set to false, then only existing Grafana users can log in with SAML.
 - `allowed_organizations` (String) List of comma- or space-separated organizations. User should be a member of at least one organization to log in.
 - `assertion_attribute_email` (String) Friendly name or name of the attribute within the SAML assertion to use as the user email.
+- `assertion_attribute_external_uid` (String) Friendly name or name of the attribute within the SAML assertion to use as the external user ID.
```

Comment:
Please change this. There's a typo and it doesn't make a reference to the SCIM provisioning.
```suggestion
- `assertion_attribute_external_uid` (String) Friendly name or name of the attribute within the SAML assertion to use as the external user ID. Only used for SCIM provisioned users.
```